backend/database/migrations/2023_07_07_091358_change_account_type_column_in_account_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class ChangeAccountTypeColumnInAccountTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement("ALTER TABLE accounts MODIFY COLUMN account_type ENUM('SAVINGS', 'SALARY', 'DEPOSIT')");
```

Comment:
use this one instead

```
Schema::table('accounts', function (Blueprint $table) {
            $table->enum('account_type', ['SAVINGS', 'SALARY', 'BILLS', 'SENDER', 'RECIPIENT'])->change();
        });
```

you may also need to install `doctrine/dbal` in order to have changes.